code/game/machinery/computer/card.dm

```diff
@@ -227,282 +291,378 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
+	switch(action)
+		// Changing basic card info
+		if("reg") // registered name on card
+			var/temp_name = reject_bad_name(input(usr, "Who is this ID for?", "ID Card Renaming", modify.registered_name), TRUE)
+			if(!modify || !temp_name)
+				visible_message("<span class='notice'>[src] buzzes rudely.</span>")
+				return FALSE
+			modify.registered_name = temp_name
+			regenerate_id_name()
+		if("account") // card account number
+			var/account_num = input(usr, "Account Number", "Input Number", null) as num|null
+			if(!scan || !modify)
+				return FALSE
+			modify.associated_account_number = clamp(round(account_num), 0, 999999)
+		if("skin")
+			if(!modify)
+				return FALSE
+			var/skin = params["skin_target"]
+			var/skin_list = is_centcom() ? get_centcom_card_skins() : get_station_card_skins()
+			if(skin in skin_list)
+				modify.icon_state = skin
+		// Changing card access
+		if("set") // add/remove a single access number
+			var/access = text2num(params["access"])
+			var/list/changable = is_centcom() ? get_all_centcom_access() + get_all_accesses() : get_all_accesses()
+			if(access in changable)
+				if(access in modify.access)
+					modify.access -= access
+				else
+					modify.access += access
+		if("grant_region")
+			var/region = text2num(params["region"])
+			if(isnull(region) || region < REGION_GENERAL || region > (is_centcom() ? REGION_CENTCOMM : REGION_COMMAND))
+				return
+			modify.access |= get_region_accesses(region)
+		if("deny_region")
+			var/region = text2num(params["region"])
+			if(isnull(region) || region < REGION_GENERAL || region > (is_centcom() ? REGION_CENTCOMM : REGION_COMMAND))
+				return
+			modify.access -= get_region_accesses(region)
+		if("clear_all")
+			modify.access = list()
+		if("grant_all")
+			modify.access = get_all_accesses()
+
+		// JOB SLOT MANAGEMENT functions
+
+		if("prioritize_job") // TOGGLE WHETHER JOB APPEARS AS PRIORITIZED IN THE LOBBY
+			var/priority_target = params["job"]
+			var/datum/job/j = SSjobs.GetJob(priority_target)
+			if(!j)
+				return FALSE
+			if(!job_in_department(j))
+				return FALSE
+			var/priority = TRUE
+			if(j in SSjobs.prioritized_jobs)
+				SSjobs.prioritized_jobs -= j
+				priority = FALSE
+			else if(SSjobs.prioritized_jobs.len < 3)
+				SSjobs.prioritized_jobs += j
+			else
+				return FALSE
+			log_game("[key_name(usr)] [priority ?  "prioritized" : "unprioritized"] the job \"[j.title]\".")
+			playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
 
-		if("wipe_my_logs")
-			if(is_authenticated(usr) && is_centcom())
-				var/delcount = SSjobs.delete_log_records(scan.registered_name, FALSE)
-				if(delcount)
-					playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
-				SSnanoui.update_uis(src)
 
-		if("wipe_all_logs")
+		if("wipe_all_logs") // Delete all records from 'records' section
 			if(is_authenticated(usr) && !target_dept)
 				var/delcount = SSjobs.delete_log_records(scan.registered_name, TRUE)
```

Comment:
Did you make this still keep a copy of logs for admins

Reply:
CC card computer can still view "deleted logs".
"Delete" functions only mark the records as hidden from non-CC computers.